modules/internal/ChapelBase.chpl

```diff
@@ -1141,7 +1136,11 @@ module ChapelBase {
   inline proc chpl__maybeAutoDestroyed(x) param return true;
 
   inline proc chpl__autoDestroy(x: object) { }
+
+  pragma "compiler generated" 
   inline proc chpl__autoDestroy(type t)  { }
+
+  pragma "compiler generated" 
   inline proc chpl__autoDestroy(x: ?t) {
     __primitive("call destructor", x);
   }
```

Comment:
Ditto - this may not necessarily fulfill the requirements of the pragma, depending on what the destructor does?